2_crosswalk_munge/src/spatial_crosswalks.R

```diff
@@ -81,10 +81,10 @@ buffer_sf_lakes <- function(out_ind, lake_ind, buffer_width){
 
   sf_donut_lakes <- sf_lakes
   for (j in 1:nrow(sf_donut_lakes)){
-    sf_donut_lakes[j, ] <- st_difference(sf_buffered_lakes[j, ], sf_lakes[j, ]) %>% select(site_id, geometry)
+    sf_donut_lakes[j, ] <- st_difference(sf_buffered_lakes[j, ], sf_lakes[j, ]) %>% dplyr::select(site_id, geometry)
   }
   sf_donut_lakes <- sf_donut_lakes %>%
-    select(site_id, geometry) %>%
+    dplyr::select(site_id, geometry) %>%
     st_transform(crs = 4326)
```

Comment:
could make the `crs` number a configuration in `lib/cfg` or something like that - it's sort of a  "magic number" right now